pegasus/sites.v3/code.org/views/learn_carousels.haml

```diff
@@ -20,12 +22,18 @@
     %button{:style=>"font-size:20px;"}
       = I18n.t(:carousel_heading_domestic)
 
-#all-tutorials{:style=>'visibility:hidden; overflow:hidden;'}
-  - categories.each do |category|
-    - id = "collection-" + category[:tag]
-    = view :learn_carousel, :heading=>category[:name], :tag=>category[:tag], :tutorials=>tutorials, :id=>id
+-if variation == 'learn'
+  #all-tutorials{:style=>'visibility:hidden; overflow:hidden;'}
+    - learn_categories.each do |category|
+      - id = "collection-" + category[:tag]
+      = view :learn_carousel, :heading=>category[:name], :tag=>category[:tag], :tutorials=>tutorials, :id=>id
 
 -if variation == 'beyond'
+  #all-tutorials{:style=>'visibility:hidden; overflow:hidden;'}
+    - beyond_categories.each do |category|
```

Comment:
looks like this is duplicate code except for beyond_categories vs. learn_categories, so we could probably just set a variable and then have the whole block just once.